### DISCUSSION TITLE
Better Icons

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#83] [Announcements]
Now the custom icons you give to tabs are more functional.
They can be filtered, and changed easily.
And now they don't replace the favicons, instead they're placed next to them.
Clicking them show a menu.
Middle clicking them removes them.
`v2211`

**Comment by N3C2L** [comment]
That's a cool addition! 
Placed next to the favicon is much better, thanks.
I'd be handsome if there were some preset icons to choose from,
like a small icon library. This small icon library could also be added to the "icons" settings page.
In case some people don't know about emoji websites to get them.

**Comment by N3C2L** [comment]
When I use the tag function to insert icons, the icons are underlined like the text tags, which often does not suit the icon. It would be very nice if the underlining could be removed for icons.

**Reply by madprops** [reply]
I don't know how to check if a text is an emoji or not. Tags are meant to be text too. So I don't see this as an issue.

**Reply by N3C2L** [reply]
Yes, you are right, it's not an issue.

**Comment by madprops** [comment]
A bit unrelated but I'm adding `Autohide Taglist` setting. When this is enabled, it will only show tags (above or below) if the item is selected. Else it hides them.

**Reply by N3C2L** [reply]
Useful, thanks.
